public/app/features/dashboard/utils/getPanelMenu.test.ts

```diff
@@ -143,4 +165,55 @@ describe('getPanelMenu', () => {
       `);
     });
   });
+
+  describe('onNavigateToExplore', () => {
+    const testSubUrl = '/testSubUrl';
+    const testUrl = '/testUrl';
+    const windowOpen = jest.fn();
+    let event: any;
+    let explore: PanelMenuItem | undefined;
+    let navigateSpy: any;
+
+    beforeAll(() => {
+      const panel = new PanelModel({});
+      const dashboard = new DashboardModel({});
+      const menuItems = getPanelMenu(dashboard, panel);
+      explore = menuItems.find(item => item.text === 'Explore');
+      navigateSpy = jest.spyOn(actions, 'navigateToExplore');
+      window.open = windowOpen;
+
+      event = {
+        ctrlKey: true,
+        preventDefault: jest.fn(),
+      };
+
+      setStore({ dispatch: jest.fn() } as any);
+    });
+
+    it('should navigate to url without subUrl', () => {
+      if (explore?.onClick) {
+        explore.onClick(event);
+      }
+
+      const openInNewWindow = navigateSpy.mock.calls[0][1].openInNewWindow;
+
+      openInNewWindow(testUrl);
+
+      expect(windowOpen).toHaveBeenLastCalledWith(testUrl);
+    });
+
+    it('should navigate to url with subUrl', () => {
+      config.appSubUrl = testSubUrl;
+
+      if (explore?.onClick) {
+        explore.onClick(event);
+      }
```

Comment:
No need for this if statement in tests I think?
```suggestion
      explore?.onClick(event);      
```